cluster/stone-prd-rh01/tenants/hcc-devprod-tenant/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ephemeral_namespace_operator_release_plan.yaml
+  - ../../../../lib/consoledot-test-pipeline
```

Comment:
This line is for the Clowder integration. Do the `ephemeral-namespace-operator` needs to be tested using itself? And if that's the case, you are still missing the `IntegrationTestScenario` that points to the Tekton pipeline with the integration